api/src/main/java/io/opentelemetry/metrics/SyncInstrument.java

```diff
@@ -26,7 +26,7 @@
  * @since 0.3.0
  */
 @ThreadSafe
-public interface InstrumentWithBinding<B extends BoundInstrument> extends Instrument {
+public interface SyncInstrument<B extends BoundInstrument> extends Instrument {
```

Comment:
Even though it's a bit longer, I prefer `SynchronousInstrument`. Also, it would be good to explain in some javadoc what "synchronous" means in this context.

Reply:
Done.